PBN RCT.Rmd

```diff
@@ -507,6 +522,31 @@ pbn_complete %>%
   group_by(name) %>%
   summarize(mean = mean(value),
             sd = sd(value))
+
+#N and percent of intervention participants with sum score >3, reflect overall perceived SSI acceptability
+included_participants_pfs %>%
+  filter(treatment_group == TRUE) %>%
+  mutate(pi_pfs_sum_scores = pi_pfs_sum / 7) %>%
+  mutate(pi_pfs_sum_scores_acceptable = pi_pfs_sum_scores > 3) %>%
```

Comment:
`>` or `>=`?

Reply:
I think I messed something up in the pre-registration. 3 is only "Neutral" but in the pre-registration, we wrote: An average score of “Agree” (3 or above on the Likert scale) on any given Program Feedback Scale item will reflect endorsement of that item

So I think we meant 4 or above / greater than 3 to have the higher acceptability threshold of "Agree" which is what we did in the pilot code too. So nothing to change in the code, but maybe we need to amend the pre-registration? https://osf.io/7ftq9

Reply:
This could also be Qualtrics automatically coding things a certain way. Either way it doesn't really matter since you pre-registered what you meant ("agree"); it's fine if that's actually coded 4, not 3

Reply:
I recoded everything in Qualtrics ahead of time. But yes all is good since we meant and got "agree"